fe/src/main/java/org/apache/doris/transaction/GlobalTransactionMgr.java

```diff
@@ -329,15 +328,20 @@ public void commitTransaction(long dbId, long transactionId, List<TabletCommitIn
                 if (!tableToPartition.get(tableId).contains(partition.getId())) {
                     continue;
                 }
-                List<MaterializedIndex> allIndices = new ArrayList<>();
-                allIndices.addAll(partition.getMaterializedIndices());
-                MaterializedIndex rollingUpIndex = null;
-                RollupJob rollupJob = null;
-                if (table.getState() == OlapTableState.ROLLUP) {
-                    rollupJob = (RollupJob) catalog.getRollupHandler().getAlterJob(tableId);
-                    rollingUpIndex = rollupJob.getRollupIndex(partition.getId());
+
+                List<MaterializedIndex> allIndices = null;
+                if (transactionState.getLoadedTblIndexes().isEmpty()) {
```

Comment:
Only stream load pu loaded indices, but dpp load does not, so that dpp load will fail during schema change process.

Reply:
Fixed